ui/src/js/storyTellingServerUI.js

```diff
@@ -155,10 +155,14 @@ fluid.defaults("sjrk.storyTelling.server.karisma.storyView", {
 
 fluid.defaults("sjrk.storyTelling.server.learningReflections.storyView", {
     gradeNames: ["sjrk.storyTelling.server.base.storyView", "sjrk.storyTelling.learningReflections.storyView"],
-    distributeOptions: {
+    distributeOptions: [{
         target: "{that menu templateManager}.options.linkConfig.templateValues.contextLinkUrl",
         record: "/storyEdit.html"
-    }
+    },
+    {
+        target: "{that menu templateManager}.options.linkConfig.templateValues.secondaryLinkUrl",
+        record: "/storyBrowse.html"
```

Comment:
Values as "/storyBrowse.html" and "/storyEdit.html" are distributed to various UI components. The architectural refactoring probably can consider to have them defined as configurable options at an upper level component and sub UI components can pull them in when needed.

Reply:
I concur!